packages/code-snippet/src/CodeSnippet.ts

```diff
@@ -41,6 +42,7 @@ export class CodeSnippetManager {
               const codeSnippet: ICodeSnippet = {
                 name: jsonCodeSnippet.name,
                 displayName: jsonCodeSnippet.display_name,
+                description: jsonCodeSnippet.metadata.description,
```

Comment:
I was having server issues running this new change when requesting code snippets, because `description` field is in `jsonCodeSnippet.description` as per our current sample files. I don't see issues having `description` inside `metadata` field instead if that's a better place for it.
Wondering if we want to change our json files schema @lresende ?

Reply:
i downloaded and used [this sample file](https://github.com/elyra-ai/elyra/pull/241#issuecomment-608605352) is there a more recent one i should be using?

Reply:
oh the schema of mine are outdated then, it's all good, thx!

Reply:
@karlaspuldaro  cool! thx for confirming